src/main/java/models/Content.java

```diff
@@ -0,0 +1,14 @@
+package models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Content {
+    public String owner;
+    public boolean share;
+    public int id;
+    public String name;
```

Comment:
You have the same fields in Dashboard class.
Maybe we can re-use them? Or it's just the same names?